DiningInsights/Pages/Favorite.cshtml.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using Microsoft.EntityFrameworkCore;
+
+namespace DiningInsights.Pages
+{
+    public class FavoriteModel : PageModel
+    {
+        private readonly AppDbContext _db;
+
+        public FavoriteModel(AppDbContext db)
+        {
+            _db = db;
+        }
+
+        public IList<Restaurant> FavoriteList { get; private set; }
```

Comment:
The naming here is confusing: we are actually return a full list of restaurant, and do the filtering job in Favorite.cshtml. So change the naming here.

Reply:
No fix.